Marsiple/App/Post Details/PostDetailsViewController.swift

```diff
@@ -10,9 +10,15 @@ import SnapKit
 
 class PostDetailsViewController: UIViewController {
     private let post: Post
-    private var comments = [Comment](repeating: Comment(email: "test.test@test.com", body: "Test comment..."), count: 10) //TODO: - Populate with real data.
+    // TODO: - Populate with real data.
+    private var comments = [Comment](repeating: Comment(email: "test.test@test.com", body: "Test comment..."), count: 10)
     private let postDetailsView = PostDetailsView.autolayoutView()
     
+    override var hidesBottomBarWhenPushed: Bool {
```

Comment:
That's the property we're looking for 🙂 

`HINT:` You don't have to override it. Setting `hidesBottomBarWhenPushed = true`'s enough.